clang/test/ClangScanDeps/modules-include-tree-implementation-transitive.c

```diff
@@ -0,0 +1,66 @@
+// REQUIRES: ondisk_cas
```

Comment:
I mentioned this to you offline, but putting here for posterity: we should extend this test to print the include-tree and check the structure of the new node. That helps clarify exactly what the test is doing.